string text
float64 scroll_speed
int16[3] text_colour
int16[3] back_colour
---